less/patternfly-react.less

```diff
@@ -1,6 +1,9 @@
 /**
   Patternfly React Specific Extensions
 */
+@import "~bootstrap/less/variables";
+@import "~patternfly/dist/less/variables";
+
```

Comment:
We don't want to do this here. Patternfly is a dependency and we assume that these are already included if the project is building with the less files. If we do this here, it will overwrite any variables the application might have already overridden (which would likely be the reason to include the less files rather than the CSS).